locales/de.json

```diff
@@ -58,7 +58,7 @@
 	"driver-request-radius.option1": "1 km",
 	"driver-request-radius.option3": "3 km, Empfohlen",
 	"driver-request-radius.option5": "5 km",
-	"driver-request-radius.should_be_valid": "Kann deine Wahl nicht verstehen. Wähle die Option 👇 oder gebe eine Zahl von 1 bis 10 ein.",
+	"driver-request-radius.should_be_valid": "Deine Auswahl ist inkorrekt. Wähle die Option 👇 oder gebe eine Zahl von 1 bis 10 ein.",
 	"global.location_button_text": "Standort senden",
 	"global.default_options_message": "Bitte wähle eine Aktion aus dem Menü unten 👇",
 	"global.default_inline_options_message": "Klicke aus die Taste 👇 oder ignoriere diese Mitteilung.",
```

Comment:
Line 68: change sentence to: "Hoppla, es gab ein Problem. Bitte versuche es erneut." This would be preferred by a native speaker to "kann das nicht verstehen", this verb isn't used in this (or any general) context.
Line 69: change to: "Hoppla, es gab ein Probem mit dem Standort. Bitte tippe auf den Button oder verwende Büroklammer-Standort um deine GPS-Daten zu senden."